src/containers/app/navigation/profile/SettingsDropdown/index.js

```diff
@@ -59,10 +58,10 @@ const SettingsDropdown = ({ logoutUser }) => {
   const menu = (
     <StyledMenu>
       <StyledMenuItem>
-        Account Profile
+        <Link to="/profile">Account Profile</Link>
       </StyledMenuItem>
-      <StyledMenuItem onClick={onSignOut} onKeyPress={onSignOut}>
-        Sign Out
+      <StyledMenuItem>
+        <Link to="/login" onClick={onSignOut}>Sign Out</Link>
```

Comment:
We have logout function in actions. We should probably utilize that here for `Sign Out`

![image](https://user-images.githubusercontent.com/12506685/84342369-41595000-ab5a-11ea-907c-c9534ddc4391.png)